config/src/keymap/key.rs

```diff
@@ -32,11 +48,9 @@ impl Default for Key {
 
 impl From<KeyEvent> for Key {
 	fn from(value: KeyEvent) -> Self {
-		let shift = matches!(value.code, KeyCode::Char(c) if c.is_ascii_uppercase());
```

Comment:
Why do we need to remove the Shift check here?

Reply:
Any char keycode now is compared without the shift modifier so it doesn't matter if uppercase characters have or don't have shift modifier. I remove it to save some execution time. Also, for it to be consistent with other characters like `~` and `?`.